tests/src/Kernel/Access/OgEntityAccessTest.php

```diff
@@ -133,6 +177,15 @@ protected function setUp() {
       ->grantPermission($this->randomMachineName())
       ->save();
 
+    $this->ogAdminRole = OgRole::create();
+    $this->ogAdminRole
+      ->setId($this->randomMachineName())
+      ->setLabel($this->randomString())
+      ->setGroupType($this->group1->getEntityTypeId())
+      ->setGroupBundle($this->groupBundle)
+      ->setIsAdmin(TRUE)
+      ->save();
```

Comment:
Oh. I see. Cool!
On 10 Jun 2016 19:05, "Amitai Burstein" <notifications@github.com> wrote:

> In tests/src/Kernel/Access/OgEntityAccessTest.php
> <https://github.com/amitaibu/og/pull/225#discussion_r66656440>:
>
> > @@ -133,6 +177,15 @@ protected function setUp() {
> >        ->grantPermission($this->randomMachineName())
> >        ->save();
> >
> > +    $this->ogAdminRole = OgRole::create();
> > +    $this->ogAdminRole
> > +      ->setId($this->randomMachineName())
> > +      ->setLabel($this->randomString())
> > +      ->setGroupType($this->group1->getEntityTypeId())
> > +      ->setGroupBundle($this->groupBundle)
> > +      ->setIsAdmin(TRUE)
> > +      ->save();
>
> Currently not, we have
>
>   protected function createPerBundleRoles($entity_type_id, $bundle_id) {    foreach ($this->getDefaultRoles() as $role_name => $default_properties) {      $properties = [        'group_type' => $entity_type_id,        'group_bundle' => $bundle_id,        'id' => $role_name,        'role_type' => OgRole::getRoleTypeByName($role_name),      ];
>
> (no is_admin). I'll create a follow up issue for this.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/amitaibu/og/pull/225/files/e67b43cb7803205e91601fdaafe60e548a43f199#r66656440>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABAUw_0l8PVfwIeJQK-rTwejZv8ov2Spks5qKadogaJpZM4Ix0vw>
> .
>